packages/compass-settings/src/components/settings/privacy.tsx

```diff
@@ -29,6 +29,8 @@ const privacyFields = [
   'autoUpdates',
   'enableMaps',
   'trackErrors',
+  'readOnly',
+  'enableShell',
```

Comment:
Should these be in the privacy section? They aren’t really privacy-related, and I think it would be fine to add a new section if we want to (although, heads up, #3588 also adds one).